appserver/packager/legal/src/main/resources/glassfish/legal/3RD-PARTY-LICENSE.txt

```diff
@@ -1024,264 +836,8 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 
 ---------------------------------------------------
-Winsw svn 49
```

Comment:
This needs to be retained. This dependency is used for configuring Windows as a service